### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by StepanPotiienko**

### CODE REVIEW

File: web/css/fonts.css [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* This file is for adding and modifying fonts */
 
+@font-face {
```
Comment (StepanPotiienko) [inline web/css/fonts.css:3]: Added new collection of Montserrat Alternates fonts, need to be removed.

**Review by StepanPotiienko — COMMENTED** [review]
Except for the font warning, everything is good to go.

**Review by 03122005pri — COMMENTED** [review]
Fine

**Review by 03122005pri — APPROVED** [review]
Otlichno

**Pull request merged**